static/app/views/performance/mobile/components/deviceClassSelector.tsx

```diff
@@ -2,18 +2,27 @@ import type {ComponentProps} from 'react';
 
 import {CompactSelect} from 'sentry/components/compactSelect';
 import {t} from 'sentry/locale';
+import {trackAnalytics} from 'sentry/utils/analytics';
 import {browserHistory} from 'sentry/utils/browserHistory';
 import {decodeScalar} from 'sentry/utils/queryString';
 import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
 import {MobileCursors} from 'sentry/views/performance/mobile/screenload/screens/constants';
+import {ModuleName} from 'sentry/views/starfish/types';
 
 interface Props {
   clearSpansTableCursor?: boolean;
+  moduleName?: ModuleName;
   size?: ComponentProps<typeof CompactSelect>['size'];
 }
 
-export function DeviceClassSelector({size = 'xs', clearSpansTableCursor}: Props) {
+export function DeviceClassSelector({
+  size = 'xs',
+  clearSpansTableCursor,
+  moduleName = ModuleName.OTHER,
```

Comment:
Is there a case that we need to default to Other?

Reply:
for `mobile-ui`